tests/index.mjs

```diff
@@ -0,0 +1,129 @@
+// eslint-disable-next-line node/file-extension-in-import
+import * as assert from 'uvu/assert';
+import {describe} from './setup/index.mjs';
+import {get} from 'httpie';
+
+const STAKING_ADDRESSES = [
+  '0x417c02150b9a31bcacb201d1d60967653384e1c6', // AVAX-ETH
```

Comment:
do you need to duplicate these constants? might need VSO here too

Reply:
Now importing from `src/utils/constants.ts` directly :)